dftimewolf/lib/collectors/workspace_audit.py

```diff
@@ -126,6 +126,15 @@ def SetUp(self,
           for.
       end_time (Optional[str]): End of the time period to return results for.
     """
+
+    # Omit '-' delimiter from the filter_expression (meeting_id) for
+    # the meet application
+    if application_name == 'meet':
+      if '-' in filter_expression:
+        filter_expression = filter_expression.replace('-', '')
+        self.logger.info(
+            'Found \'-\' delimiter in the meeting_id and removed it!')
```

Comment:
```suggestion
            "Found '-' delimiter in the meeting_id and removed it!")
```

is slightly more readable. You're allowed to infringe the always-use-the-same-quote-type rules when you are including quotes in a string :)